index.js

```diff
@@ -1,10 +1,24 @@
 'use strict';
 var setImmediate = require('set-immediate-shim');
 var hasFlag = require('has-flag');
+var relative = require('path').relative;
 var serializeError = require('serialize-error');
 var Runner = require('./lib/runner');
+var log = require('./lib/logger');
 var runner = new Runner();
 
+// check if the test is being run without AVA cli
+var isForked = typeof process.send === 'function';
```

Comment:
Probably clearer to just check if the file was executed directly: `require.main === module`